holoviews/core/spaces.py

```diff
@@ -495,6 +495,13 @@ def __call__(self, *args, **kwargs):
         return ret
 
 
+class OverlayCallable(Callable):
+    """
+    A Callable subclass specifically meant to indicate that the Callable
+    represents an overlay operation between two objects.
+    """
```

Comment:
Is this a class users should ever make use of directly? I would have no idea whether to use it or not based on the current docstring.